packages/p2p/src/stores/order-store.js

```diff
@@ -193,6 +200,21 @@ export default class OrderStore {
         });
     }
 
+    handleDateChange(date_values, { date_range } = {}) {
+        const { from, to, is_batch } = date_values;
+
+        this.filtered_date_range = date_range;
+
+        if (from) {
+            this.date_from = toMoment(from).unix();
+        } else if (is_batch) {
+            this.date_from = null;
+        }
+
+        if (to) this.date_to = toMoment(to).unix();
+        this.loadMoreOrders({}, true);
+    }
```

Comment:
can we handle this is in the component?

Reply:
@nada-deriv did you missed this? 👀

Reply:
no. had a discussion with @farrah-deriv  regarding this. haven't come to a conclusion yet.

Reply:
let us know if you can move this into the component 🙏

Reply:
as discussed, will be keeping the implementation as it is for now.